README.md

```diff
@@ -30,6 +30,19 @@ npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
 
 ---
 
+## Supported networks
+
+| Network | Chain ID | Type       |
+|---------| -------- | ---------- |
+| mainnet | 1        | Production |
+| ropsten | 3        | Test       |
+| rinkeby | 4        | Test       |
+| goerli  | 5        | Test       |
+| polygon | 137      | Production |
+| mumbai  | 80001    | Test       |
```

Comment:
Now that we aren't restricted to the names ethers have, we are free to use whatever names we like! I changed it to `polygon` and `mumbai` here, but feel free to change it to anything else you think will be better.